libcudacxx/examples/nv_target.cu

```diff
@@ -60,16 +60,19 @@ Static true/false values for fallbacks or user manipulation
 
 //=======================================================================================================================
 // NV_IF_ELSE_TARGET(query, true statement, false statement)
-__host__ __device__ int my_popc(unsigned int v) {
+__host__ __device__ int my_popc(unsigned int v)
+{
   // NV_IF_ELSE_TARGET accepts three arguments, a query and two statement.
   // Here we check if we're compiling for device code. This function acts as a backend for both CUDA and host CPU popc.
   NV_IF_ELSE_TARGET(
     NV_IS_DEVICE,
-      return __popc(v);,        // Is false, use CUDA intrinsic
-      return __builtin_popc(v); // Is host, use GCC builtin
+    return __popc(v);
+    , // Is false, use CUDA intrinsic
```

Comment:
Seems like a typo to have the comma here.
```suggestion
    return __popc(v); // Is false, use CUDA intrinsic
```

Reply:
That one is actually valid. I changed the comments. This is inside a NV_IF_TARGET and the comma separates the different conditionals